conf/messages.en

```diff
@@ -143,6 +143,19 @@ newContainerIdentificationNumber.error.length = The identification number must b
 newContainerIdentificationNumber.error.characters = The identification number must only include letters a to z and numbers 0 to 9
 newContainerIdentificationNumber.error.alreadyEntered = This identification number has already been submitted. Enter a unique container identification number
 
+containerIdentificationNumber.NormalMode.title = What is the container identification number?
+containerIdentificationNumber.CheckMode.title = What is the new container identification number?
+containerIdentificationNumber.NormalMode.heading = What is the container identification number?
+containerIdentificationNumber.CheckMode.heading = What is the new container identification number?
+containerIdentificationNumber.NormalMode.paragraph=
```

Comment:
empty message key

Reply:
It's on purpose, For Normal mode there's no paragraph in the content pages - also checked with Gavin if this is an expected behaviour? he mentioned it's correct.